bibliography-sage.bib

```diff
@@ -2495,7 +2495,7 @@ @article{GunawanScrimshaw2017
   date         = {2017},
   journaltitle = {Sém. Lothar. Combin.},
   pages        = {Art. 47,12},
-  title        = {Realization of Kirillov-Reshetikhin crystals {$B^{1,s}$} for {$\widehat{\mathfrak{sl}}_n$} using Nakajima monomials},
+  title        = {Realization of Kirillov-Reshetikhin crystals {$B^{1,s}$} for $\widehat{\mathfrak{sl}}_n$ using Nakajima monomials},
```

Comment:
![Screenshot from 2022-02-25 17-35-04](https://user-images.githubusercontent.com/207405/155752256-3ba47409-1bc9-48e0-a41c-c50fa73d00a5.png)

Reply:
By the way, this was exactly like it looked like befor the change.

Reply:
hmm, maybe something with mathfrac is wrong. weird.